main.go

```diff
@@ -325,15 +327,17 @@ func main() {
 		ext := filepath.Ext(apkFile)
 		baseName := filepath.Base(apkFile)
 		baseName = strings.TrimSuffix(baseName, ext)
+		fileName := baseName + ext
+
+		log.Printf("Copying %s --> %s", apkFile, filepath.Join(configs.DeployDir, fileName))
 
 		deployPth, err := findDeployPth(configs.DeployDir, baseName, ext)
 		if err != nil {
-			failf("Failed to create apk deploy path, error: %s", err)
+			failf("Failed to create deploy path for %s, error: %s", fileName, err)
```

Comment:
NORE: The `, error: ` part is not the current convention for error messages.

Reply:
Updated